plugins.d/python.d.plugin

```diff
@@ -16,6 +17,7 @@ from re import sub
 from sys import version_info, argv
 from time import sleep
 
+GC_COLLECT_EVERY = 300
```

Comment:
I suggest to put an option in `python.d.conf` for this so that people can run it more or less frequently, and allow to be disabled.